### DISCUSSION TITLE
GeoJSON importer

### DISCUSSION THREAD

**Discussion by hylkevds** [opengeospatial/sensorthings#144] [Show and tell]
Hi All,

I recently ran into the issue that I needed to import a bunch of features from a GeoJSON file, and create Locations and Things from them. So I wrote a GeoJSON importer: https://github.com/hylkevds/FROST-GeoJsonImporter
It's a Java GUI application that can be configured to create a Location, and optionally a Thing, for each feature in a feature collection.

Feedback is warmly welcomed :)

**Comment by liangsteve** [comment]
@liangsteve @hylkevds @MarcusAlzona to make a resource page on GitHub, so we can keep a list of resources.
